tests/GistConfigTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Gistlog\Gists\GistConfig;
+
+class GistConfigTest extends TestCase
+{
+    use GistFixtureHelpers;
+
+    /** @test */
+    public function it_can_be_created_from_github_api_data()
+    {
+        $githubGist = $this->loadFixture('8f5ea4d44dbc5ccb77a3.json');
+
+        $config = GistConfig::fromGitHub($githubGist);
+
+        $this->assertEquals("A test preview.", $config['preview']);
+        $this->assertEquals("05-15-2015", $config['published_on']);
```

Comment:
Good call on this

http://symfony.com/legacy/doc/reference/1_3/en/02-YAML#chapter_02_sub_dates

Specifies that `YAML` dates must be in this format in order to be recognized as dates. Only issue this introduces is that the Symfony `YAML` parser now returns a timestamp instead of the actual date string when parsing through said date.